free-courses-en.md

```diff
@@ -224,6 +232,11 @@
 * [Principles of Reactive Programming](https://www.coursera.org/course/reactive)
 
 
+### SQL
+
+* [Code School - Try SQL](https://www.codeschool.com/courses/try-sql)
```

Comment:
Could you please move this link to `free-programming-interactive-tutorials-en.md`?